lighthouse-cli/index.js

```diff
@@ -77,6 +78,13 @@ if (cli.flags.listAllAudits) {
   process.exit(0);
 }
 
+if (cli.flags.listTraceCategories) {
+  const categories = lighthouse.traceCategories;
+
+  log.info('All trace categories required by lighthouse:', categories.join(', '));
```

Comment:
Since we want this to be machine readable, i would default to be more structured, like..

```js
process.stdout.write(JSON.stringify(categories))
```

(with no logging statement that you'd have to parse out)